kafka-connect-upload-source/src/main/java/org/radarbase/connect/upload/converter/oxford/WearableCameraConverterFactory.kt

```diff
@@ -0,0 +1,33 @@
+package org.radarbase.connect.upload.converter.oxford
+
+import org.radarbase.connect.upload.api.SourceTypeDTO
+import org.radarbase.connect.upload.converter.ConverterFactory
+import org.radarbase.connect.upload.converter.FileProcessorFactory
+import org.radarbase.connect.upload.converter.LogRepository
+import org.radarbase.connect.upload.converter.ZipFileProcessorFactory
+import org.radarbase.connect.upload.io.SftpFileUploader
+import java.net.URL
+import java.nio.file.Paths
+
+class WearableCameraConverterFactory : ConverterFactory {
+    override val sourceType: String = "oxford-wearable-camera"
+
+    override fun fileProcessorFactories(settings: Map<String, String>, connectorConfig: SourceTypeDTO, logRepository: LogRepository): List<FileProcessorFactory> {
+        val sourceConfig = checkNotNull(connectorConfig.configuration)
+        val credentials = SftpFileUploader.SftpCredentials(
+                host = sourceConfig.getValue("host"),
```

Comment:
This would be part of the `map` configuration for this source-type. When we release, we should also add it as part of the documentation saying which keys and values are expected.